src/app/app.js

```diff
@@ -29,7 +30,8 @@ const app = (config) => {
     init(accordion, document.querySelector('.footer__links'));
     init(reminder, document.querySelector('.js-reminder'));
     init(rateProduct, document.querySelector('.rate-product'));
-    init(history, document.querySelector('.history-modal'));
+    // init(history, document.querySelector('.history-modal'));
+    init(loadDataModal, document.querySelector('.history-modal'));
```

Comment:
just the naming, as was in Artem's guideline **action + high context + low context** => `loadModalData` or `getModalData`